apps/docs/components/Navigation/NavigationMenu/NavigationMenu.constants.ts

```diff
@@ -172,6 +172,11 @@ export const GLOBAL_MENU_ITEMS: GlobalMenuItems = [
       label: 'Resources',
       menuItems: [
         [
+          /*           {
+            label: 'Troubleshooting',
+            icon: 'contributing',
```

Comment:
@fsansalvadore any suggestions for icons? currently contributing uses the wrench icon, which probably should be transferred to troubleshooting, but then I'm not sure what to use for contributing

Reply:
`Contributing` could either be a simple _heart_ or a _"branching"_ icon, maybe?

<img width="26" alt="Screenshot 2024-09-11 at 14 24 08" src="https://github.com/user-attachments/assets/c2ecc4e1-ace4-46f5-bae3-3221b3cab02b">

And I agree troubleshooting would be the wrench.